src/controller/python/chip/native/StackInit.cpp

```diff
@@ -56,11 +56,13 @@ extern "C" void pychip_native_init()
     }
 #endif // CHIP_DEVICE_LAYER_TARGET_LINUX && CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
+#ifndef CONFIG_DEVICE_LAYER
     err = chip::DeviceLayer::PlatformMgr().InitChipStack();
```

Comment:
If CONFIG_DEVICE_LAYER is not defined, chip::DeviceLayer would not exist.

python is only built on platforms that have a device layer.